client/src/main/java/io/pravega/client/stream/impl/PravegaDefaultCredentials.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PravegaDefaultCredentials implements PravegaCredentials {
```

Comment:
Is a user supposed to instantiate this class? It's in .impl but I don't see a factory method for this in the public. Perhaps this is meant to be extracted from the config?

Reply:
The user will use this if they want to depend on Pravega's implementation of `PravegaCredentials` interface. Do you have a recommendation for better place for this?